frontend/src/components/dashboard/aliases/AddressPickerModal.tsx

```diff
@@ -1,12 +1,14 @@
 import {
+  ChangeEventHandler,
+  FocusEventHandler,
   FormEventHandler,
   ReactElement,
   ReactNode,
   useRef,
   useState,
 } from "react";
 import Link from "next/link";
-import { useLocalization } from "@fluent/react";
+import { ReactLocalization, useLocalization } from "@fluent/react";
```

Comment:
question (non-blocking): Am I understanding correctly that we import the `ReactLocalization` class to use in a function rather than using the `useLocalization` hook like we do in the component?

If so, that makes me think that `getAddressValidationMessage` function belongs in `frontend/src/functions` rather than here in `frontend/src/components`?

Reply:
There's a small nuance there in that it's technically not a class I'm importing, but that class's type (i.e. you won't see it in the final JS bundle), but yes, it's used to indicate that the function expects an instance of that class because hooks only work in components.

So yeah, there's certainly something to be said for moving it into functions. The reason I kept it in this file is that I didn't expect the function to be re-used in different files; the only reason I extracted it into a separate function in the first place is so that I could isolate the interesting behaviour for a unit test.